src/components/NewConversationDialog/NewConversationDialog.vue

```diff
@@ -227,6 +228,11 @@ export default {
 				|| this.conversationName.length > CONVERSATION.MAX_NAME_LENGTH
 				|| this.newConversation.description.length > CONVERSATION.MAX_DESCRIPTION_LENGTH
 		},
+
+		isFilled() {
+			return JSON.stringify(this.newConversation) !== JSON.stringify(NEW_CONVERSATION)
+				|| this.listable !== CONVERSATION.LISTABLE.NONE || this.isAvatarEdited
```

Comment:
description? password?

but I think name should be enough, most people will start with filling that one

Reply:
evrything you don't see here is included in `this.newConversation` object